bundle/manifests/opentelemetry-operator.clusterserviceversion.yaml

```diff
@@ -487,9 +571,9 @@ spec:
         serviceAccountName: opentelemetry-operator-controller-manager
     strategy: deployment
   installModes:
-  - supported: true
+  - supported: false
     type: OwnNamespace
-  - supported: true
+  - supported: false
```

Comment:
what do these flags do?

Reply:
IIRC It's for the OLM to allow watching of single or multiple namespaces. This was required for introducing the new version.

Reply:
https://olm.operatorframework.io/docs/tasks/install-operator-with-olm/

https://docs.openshift.com/container-platform/4.15/operators/understanding/olm/olm-understanding-operatorgroups.html#olm-operatorgroups-membership_olm-understanding-operatorgroups

https://docs.openshift.com/container-platform/4.15/operators/operator_sdk/osdk-generating-csvs.html 

![Screenshot of Defining cluster service versions (CSVs) - Developing Operators _ Operators _ OpenShift Container Platform 4 15](https://github.com/open-telemetry/opentelemetry-operator/assets/5618424/3496674c-9810-4ed3-8dde-c592e110ae6d)

Reply:
I have noted this in the release notes